workflow/controller/retry_tweak.go

```diff
@@ -11,13 +11,30 @@ import (
 // RetryTweak is a 2nd order function interface for tweaking the retry
 type RetryTweak = func(retryStrategy wfv1.RetryStrategy, nodes wfv1.Nodes, pod *apiv1.Pod)
 
-// FindRetryNode locates the closes retry node ancestor to nodeID
-func FindRetryNode(nodes wfv1.Nodes, nodeID string) *wfv1.NodeStatus {
+// FindRetryNode locates the closes retry node ancestor to nodeID.
```

Comment:
couple of typos: 
1. Line 14: "closes" -> "closest"
2. Line 17: "no set" -> "not set"